terratest/src/test/pega/deployment_utility.go

```diff
@@ -334,6 +373,18 @@ func VerifySearchService(t *testing.T, helmChartPath string, options *helm.Optio
 	require.Equal(t, searchServiceObj.Spec.Ports[0].TargetPort, intstr.FromInt(9200))
 }
 
+// VerifyConstellationService - Verifies constellation service deployment used by constellation pod with the values as provided in default values.yaml
+func VerifyConstellationService(t *testing.T, helmChartPath string, options *helm.Options) {
+
+	constellationService := helm.RenderTemplate(t, options, helmChartPath, []string{"charts/constellation/templates/clln-service.yaml"})
+	var constellationServiceObj k8score.Service
+	helm.UnmarshalK8SYaml(t, constellationService, &constellationServiceObj)
+	require.Equal(t, constellationServiceObj.Spec.Selector["app"], "constellation")
+	// require.Equal(t, constellationServiceObj.Spec.Ports[0].Protocol, "TCP")
+	require.Equal(t, constellationServiceObj.Spec.Ports[0].Port, int32(3000))
```

Comment:
Do we have any reason to use port 3000 on the service? If not, let's use standard http/https ports.

Reply:
This was explained by the c11n team as a way to keep the service separate from infinity when deployed as a sidecar (still supported).